python/ray/train/v2/_internal/execution/worker_group/protocol.py

```diff
@@ -0,0 +1,32 @@
+from abc import abstractmethod
+from typing import Dict, Protocol
+
+
+class PolicyHandledStatus(Protocol):
```

Comment:
This naming would probably be more consistent:

```suggestion
class WorkerGroupStatus(Protocol):
```